cvat-ui/src/components/annotation-page/review/issues-aggregator.tsx

```diff
@@ -38,6 +38,7 @@ export default function IssueAggregatorComponent(): JSX.Element | null {
     const issuesResolvedHidden = useSelector((state: CombinedState): boolean => state.review.issuesResolvedHidden);
     const canvasInstance = useSelector((state: CombinedState) => state.annotation.canvas.instance);
     const canvasIsReady = useSelector((state: CombinedState): boolean => state.annotation.canvas.ready);
+    const annotationsZLayer = useSelector((state: CombinedState): number => state.annotation.annotations.zLayer.cur);
```

Comment:
I will suggest to replace 13 `useSelector` calls with one `useSelector` call, returning an object and using `shallowEqual` callback for comparison